src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/FlipViewItem_themeresources.xaml

```diff
@@ -27,13 +27,16 @@
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="FlipViewItem">
-                    <ContentPresenter Content="{TemplateBinding Content}"
+                    <!-- Uno specific - Explicitly setting ContentTemplateSelector -->
```

Comment:
@ajpinedam Analogous change needs to be done in:
- src/Uno.UI.FluentTheme.v1/Resources/Version1/PriorityDefault/FlipViewItem_themeresources.xaml
- src/Uno.UI/UI/Xaml/Controls/FlipView/FlipViewItem_themeresources.xaml

Reply:
Thanks @MartinZikmund ! Updated